notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>After showing the raw images using matplotlib above, could you also use Jdaviz Imviz to show the images? That is a great way to show how users can use another MAST tool!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>